### DISCUSSION TITLE
use vertical position to denote time of day

### DISCUSSION THREAD

**Discussion by scaramouche1** [kiwanami/emacs-calfw#175] [Ideas]
Currently, events within a day are displayed one after the other, sorted by time of day.  For instance:

```
08:00 Meeting A
12:00 Meeting B
13:00 Meeting C
18:00 Meeting D
```

It would be nice if the vertical position was roughly related to the time of the day, so that, e.g., a noon event is shown near the middle of the day, and a 6pm event near the end.  For example, the previous example could look as:

```
08:00 Meeting A


12:00 Meeting B
13:00 Meeting C


18:00 Meeting D
```

I believe implementing this would make calfw more similar to other electronic calendars, and also more useful, as vertical positioning provides a fast way to understand the day's events.

Many thanks for the great software!

**Comment by ragloo** [comment]
+1 on this feature

May I suggest also to shift or add the time information in a column on the left of the first one such as Google calendar for example.

As we are talking about the vertical position of the events, I have few different Google calendars displayed at the same time in calfw.  They are first listed for one calendar then the other one so on. With this behavior you have  sometimes events that are listed after the first displayed calendar whereas they start earlier in the day.

Anyway, many many thanks for this excellent package ... just awesome

**Reply by haji-ali** [reply]
> May I suggest also to shift or add the time information in a column on the left of the first one such as Google calendar for example.

This is better served by `calfw-blocks`.



> As we are talking about the vertical position of the events, I have few different Google calendars displayed at the same time in calfw. They are first listed for one calendar then the other one so on. With this behavior you have sometimes events that are listed after the first displayed calendar whereas they start earlier in the day.

I think this is a reasonable expectation. Added as an issue #176

**Comment by haji-ali** [comment]
I believe using vertical position to denote time of day is better served by `calfw-blocks`.
